token_dispenser/configuration.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+DEFAULT_LAUNCHPAD_GETTOKEN_URL:str =  "https://api.launchpad.nasa.gov/icam/api/sm/v1/gettoken"
+
+# Launchpad token api url
+LAUNCHPAD_GETTOKEN_URL: str = os.getenv('LAUNCHPAD_GETTOKEN_URL', DEFAULT_LAUNCHPAD_GETTOKEN_URL)
+# The secret-id point to the Launchpad pfx password
+LAUNCHPAD_PFX_PASSWORD_SECRET_ARN:str = os.getenv('LAUNCHPAD_PFX_PASSWORD_SECRET_ARN', '')
+# The bucket will launchpad.pfx is stored. Ex. my-sndbx-bucket
+LAUNCHPAD_PFX_FILE_S3_BUCKET = os.environ.get('LAUNCHPAD_PFX_FILE_S3_BUCKET')
```

Comment:
Shouldn't we just use `environ['LAUNCHPAD_PFX_FILE_S3_BUCKET']` so that we get an immediate key error if the environment variable doesn't exist at runtime?